src/components/ImportRecordsFlow/readme.md

```diff
@@ -237,3 +236,102 @@ const ImportRecordsFlowModal = () => {
 
 ```
 
+# ImportRecordFlow using validateRecordsFn
+##### The `validateRecordsFn` prop is a function to validate the record before import it. This function will be invoked on each record of the CSV returning an object with the errors found in a record on each field. If the object doesn't have properties then the record is valid.
+
+```js
+import React from 'react';
+import { ImportRecordsFlow, Button } from 'react-rainbow-components';
+
+const containerStyles = { height: 360 };
+
+const schema = {
+    collection: 'contacts',
+    attributes: {
+        firstName: {
+            type: String,
+            required: true,
+        },
+        email: {
+            type: String,
+            required: true,
+        },
+        phone: {
+            type: String,
+            required: true,
+        },
+        lastName: String,
+    },
+};
+
+function isPhoneValid(phone) {
+    const phoneRegex = /^\d{10}$/;
+    return phoneRegex.test(phone);
+};
+
+function isEmailValid (email) {
+    const emailRegex = /^[\w-.]+@([\w-]+\.)+[\w-]{2,4}$/;
+    return emailRegex.test(email);
+};
+
+function validateRecords(record) {
+    const error = {};
+    if (record.firstName === undefined ) {
+        error.firstname = 'Name is required';
+    }
+    if (record.phone !== undefined && !isPhoneValid(record.phone)) {
+        error.phone = 'Phone is not valid';
+    }
+    if (!isEmailValid(record.email)) {
+        error.email = 'Email is not valid';
+    }
+    return error;
+}
+
+class ImportRecordsFlowModal extends React.Component {
```

Comment:
using class in react is an old way of creating components, instead, let's use functional components with hooks for all new examples, there are old examples with class because at the time of writing them, there was no hooks yet but new examples should be created as functional components/